src/scripts/farming/Farming.ts

```diff
@@ -1146,6 +1149,27 @@ class Farming implements Feature {
         this.resetAuras();
     }
 
+    /**
+     * Handles using the Mulch Shovel to remove mulch from a plot
+     * @param index The plot index
+     */
+    public shovelMulch(index: number): void {
+        const plot = this.plotList[index];
+        if (!plot.isUnlocked) {
+            return;
+        }
+        if (this.mulchShovelAmt() <= 0) {
+            return;
+        }
+
+        if (plot.clearMulch()) {
+            GameHelper.incrementObservable(this.mulchShovelAmt, -1);
+            //GameHelper.incrementObservable(App.game.statistics.totalShovelsUsed, 1);
```

Comment:
I'm still unsure about this, should I increment it ? Should I add another variable to statistics, this time specific to mulch shovel ?

Reply:
Hmm, I think either option works.
A new statistic, or just use the existing shovel statistic.